generator/internal/sidekick/refreshall.go

```diff
@@ -25,7 +25,7 @@ import (
 	"sync"
 )
 
-func RefreshAll(rootConfig *Config, cmdLine *CommandLine) error {
+func refreshAll(rootConfig *Config, cmdLine *CommandLine) error {
```

Comment:
unrelated: now that I see all the commands together, this one is missing a comment.  I will add that later.